crates/stc_ts_type_checker/tests/conformance/types/typeRelationships/typeInference/contextualSignatureInstantiation.ts.stats.rust-debug

```diff
@@ -1,6 +1,6 @@
 Stats {
-    required_error: 0,
-    matched_error: 6,
+    required_error: 3,
```

Comment:
This is a regression, and to fix it we should not allow assignment to parameters